app/retail/templates/shared/footer_scripts.html

```diff
@@ -15,35 +15,34 @@
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 {% endcomment %}
-{% load static %}
+{% load static compress %}
 <script type="text/javascript">
   var static_url = "{{ STATIC_URL }}";
   var media_url = "{{ MEDIA_URL }}";
   document.contxt = {{ json_context|safe }};
 </script>
-<script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
-<script src="{% static "v2/js/lib/jquery.js" %}"></script>
 {% if user.is_authenticated and env == 'prod' %}
-  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
-{% elif user.is_authenticated %}
-  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vue"></script> 
 {% endif %}
+
 {% include 'shared/sentry.html' %}
-<script src="{% static "v2/js/lib/popper.min.js" %}"></script>
-<script src="{% static "v2/js/lib/bootstrap.min.js" %}" crossorigin="anonymous"></script>
 <script>
   const bootstrapModal = $.fn.modal.noConflict()
   $.fn.bootstrapModal = bootstrapModal
   const bootstrapTooltip = $.fn.tooltip.noConflict()
   $.fn.bootstrapTooltip = bootstrapTooltip;
 
 </script>
+<script src="{% static "v2/js/lib/jquery.js" %}"></script>
 
+{% compress js %}
+<script src="{% static "v2/js/lib/popper.min.js" %}"></script>
+<script src="{% static "v2/js/lib/bootstrap.min.js" %}" crossorigin="anonymous"></script>
+<script src="{% static "jsi18n/en/djangojs.js" %}"></script>
+<script src="{% static "v2/js/lib/vue.js" %}"></script>
 <script src="{% static "v2/js/lib/jquery.cookie.js" %}"></script>
-{% if not slim %}
-  <script src="{% static "v2/js/lib/jquery-ui.js" %}"></script>
-  <script src="{% static "v2/js/lib/tooltip.js" %}"></script>
-{% endif %}
+<script src="{% static "v2/js/lib/jquery-ui.js" %}"></script>
```

Comment:
this will break a lot of places using modals and popovers from bootstrap, that the main reason of slim, also as we are trying to totally remove this from the product